test/bigtest/tests/title-search-test.js

```diff
@@ -80,19 +80,12 @@ describe('TitleSearch', () => {
   });
 
   it('filter accordions should be collapsed by default', () => {
-    expect(TitleSearchPage.tagsFilterAccordion.isOpen).to.be.false;
+    expect(TitleSearchPage.tagsSection.tagsAccordion.isOpen).to.be.false;
     expect(TitleSearchPage.typeFilterAccordion.isOpen).to.be.false;
     expect(TitleSearchPage.sortFilterAccordion.isOpen).to.be.false;
     expect(TitleSearchPage.selectionFilterAccordion.isOpen).to.be.false;
   });
 
-  it('should display tags multiselect disabled by default', () => {
-    expect(TitleSearchPage.tagsSection.tagsMultiselectIsDisabled).to.be.true;
-  });
-
-  it('search by tags tags checkbox should be not checked', () => {
-    expect(TitleSearchPage.tagsSection.tagsCheckboxIsChecked).to.be.false;
-  });
```

Comment:
Just curious -- are these removed since the tags accordion should be opened first before checking state? and the tests are redundant with the same in package tests?

Reply:
Yeap, the  accordion is closed and the test is redundant.